client/_tests_/components/__snapshots__/footer.test.tsx.snap

```diff
@@ -0,0 +1,114 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`footer test is redendring properly footer component  1`] = `
+<footer
```

Comment:
Isn't possible just check if the componet was called? Otherwise this test gonna fail for every small change  :/

Reply:
yea could be, you saying now i think is better to check if it was called then if it contains something

Reply:
but it returns a html component, why don't use snapshot?
ok it will fail every change, but if is a html component how i will know is right?

Reply:
do you think is better to create a snapshot for layout and just check if header and footer was called?
because footer and header literally repeats them self's inside layout snapshot

Reply:
ok got it

Reply:
hey i thinking about to drop just 1 test, stay with layout's tests to check if the others components are right, what u think?

Reply:
hey do you think add one more issue int this pr?? 
to add tabler to remove that svg codes

Reply:
ok then

Reply:
the tests are done.